### DISCUSSION TITLE
No more GitHub release on the repository

### DISCUSSION THREAD

**Discussion by Klaitos** [swc-project/swc#7567] [Q&A] [answered]
Hello,

Before [v1.3.61](https://github.com/swc-project/swc/releases/tag/v1.3.61) tag there was one tag = one GitHub releaase, with precompiled binaries for each architecture.
Since 3 weeks now, there is no more release but only new tag.

Is there a change on how swc core is handled ? We used to download the binary from the release page in our CLI.

Thanks

**Comment by kdy1** [comment, marked as answer]
It's a regression caused by https://github.com/swc-project/swc/pull/7481

**Reply by kdy1** [reply]
It's not intentional, though

**Reply by Klaitos** [reply]
Great ! waiting for the fix then.
Thanks.

**Reply by Klaitos** [reply]
Any chance to get an official release soon ?

**Comment by alexeagle** [comment]
Looks like it was fixed two days ago https://github.com/swc-project/swc/releases/tag/v1.3.75
